management/nodes/reboot/setup_steps.README

```diff
@@ -0,0 +1,41 @@
+### Prerequisites
+
+Follow the setup steps from <v-jiba@microsoft.com> to get a triple-boot machine</br>
```

Comment:
You might make a note here that the paths  (like mount points) in the guide are dependent on that setup logic, so would need to be changed if we had a slightly different boot setup.